rust/datafusion/src/execution/context.rs

```diff
@@ -140,10 +142,10 @@ impl ExecutionContext {
 
     /// Creates a logical plan
     pub fn create_logical_plan(&mut self, sql: &str) -> Result<LogicalPlan> {
-        let ast = DFParser::parse_sql(sql)?;
+        let statements = DFParser::parse_sql(sql)?;
 
-        match ast {
-            DFASTNode::ANSI(ansi) => {
+        match &statements[0] {
```

Comment:
I think we should return an `Err` result here if `statements.len() != 1`.

Reply:
Well spotted. Fixed